.github/workflows/automationTemplate.yaml

```diff
@@ -53,28 +53,18 @@ jobs:
       - name: Run SDK Automation
         run: dotnet run --project build/tools/Automation/Automation.csproj ${RELEASE_DATE} ${{ github.workspace }}
 
-      - name: Check if any git changes exist.
-        run: |
-          changes=$(git diff-index --name-only HEAD --)
-          echo ${changes}
-          if [ -z "${changes}" ]; then
-              echo "No git changes to commit. Cancelling workflow..."
-              exit 1
-          fi
-          git status
-
       - name: Commit new ${{ env.PLATFORM_NAME_CAMEL_CASE }} SDK versions
         run: |
-          git status
+          set -x
           git config user.name 'Oryx Team'
           git config user.email 'oryx@microsoft.com'
+          
           git add -A
-          git commit -m "Add new SDK versions"
+          git commit -m "Add new SDK versions" | true
+          gitCommitExitCode=${PIPESTATUS[0]}
+          [[ $gitCommitExitCode = 0 ]] && cancelWorkflow=false || cancelWorkflow=true
```

Comment:
Hope I'm not misunderstanding the code: should this line be an `if / else` block? It feels a bit confusing to make this in a one liner that looks like a condition.

Reply:
This one is a bash ternary operator but I can update to be more explicit.

Reply:
Updated 😃

Reply:
OK my bad, I didn't know about this structure in bash. Thanks for updating!